python/tests/helpers/io.py

```diff
@@ -8,6 +8,8 @@
 from contextlib import contextmanager
 from tempfile import NamedTemporaryFile, TemporaryDirectory
 
+from aac.util import new_working_dir
```

Comment:
I believe the only use of `new_working_dir` is here an in some other tests, we should move it from util.py to here.